olive/model/hf_utils.py

```diff
@@ -370,14 +382,9 @@ def get_hf_model_io_config(model_name: str, task: str, feature: Optional[str] =
     return io_config
 
 
-def get_hf_model_dummy_input(
-    model_name: str,
-    task: str,
-    feature: Optional[str] = None,
-    trust_remote_code: Optional[bool] = None,
-):
-    model_config = get_onnx_config(model_name, task, feature)
-    tokenizer = AutoTokenizer.from_pretrained(model_name, trust_remote_code=trust_remote_code)
+def get_hf_model_dummy_input(model_name: str, task: str, feature: Optional[str] = None, **kwargs):
+    model_config = get_onnx_config(model_name, task, feature, **kwargs)
```

Comment:
It might not be related with this PR. It looks weird to call `get_onnx_config` in `get_hf_model_dummy_input`

Reply:
Same comment as above.